.github/workflows/pull_request.yml

```diff
@@ -7,33 +7,62 @@ on:
     types: [opened, synchronize]
 
 jobs:
+  lint-commits:
```

Comment:
Could we combine this into the verify job?

Reply:
👍 Updated